cfme/tests/cli/test_appliance_console.py

```diff
@@ -1293,8 +1292,41 @@ def test_appliance_console_static_dns():
         caseimportance: low
         caseposneg: negative
         initialEstimate: 1/6h
+        testSteps:
+            1. configure static ipv4
+            2. reconfigure static ipv4
+            3. configure static ipv6
+            4. reconfigure static ipv6
+        expectedResults:
+            1.
+            2. check default secondary DNS is listed or not while entering  DNS
+            3.
+            4. check default secondary DNS is listed or not while entering  DNS
     """
-    pass
+    appliance = temp_appliance_preconfig_long
+    command_set = ("ap", RETURN)
+    result = appliance.appliance_console.run_commands(command_set, timeout=30)
+    first_screen = result[0].split("\n")
+    secondary_dns_regex = r'(Secondary DNS:\s*)(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})'
```

Comment:
Can you move this to a constant (outside of the test function).

Reply:
Done